bears/python/PEP8Bear.py

```diff
@@ -28,6 +28,7 @@ def run(self, filename, file,
             local_pep8_config: bool = False,
             ):
         """
+
```

Comment:
Why did you add a new line here?

I think this issue has nothing to do with indentation. The only problem I can see here is the absence of a class docstring which is currently included in the function docstring. They should be separated. Idk if there's a separate issue for this. 

Also, stop closing your pull requests repeatedly.

Reply:
But there is no function docstring in PEP8Bear.py
Can u please tell me what I actually I need to do to resolve this issue

Reply:
~I think this issue has nothing to do with indentation.~ It might have. Start by looking at other bears on how they have indented it. 

And go through the [Git Tutorial](http://api.coala.io/en/latest/Developers/Git_Basics.html) so you don't have to close a PR again. You've already did that 7 times in the past 3 days.

Reply:
But I have seen other bears files and I find same indentation like in PEP8Bear.py file
No change is there 
I m not getting as  there is no indentation problem in the file PEP8Bear.py